libeos-payg/provider.c

```diff
@@ -375,6 +375,7 @@ epg_provider_shutdown_finish (EpgProvider   *self,
  * @self: an #EpgProvider
  * @delta: the amount by which the clock changed in seconds, which can be
  * positive or negative
+ * @now_secs: the time that the clock jump occurred, in unix seconds
```

Comment:
Nitpick: Is that time given in the frame of reference of the old clock, or the new one?

Otherwise LGTM, and you could probably go ahead and merge it once you’ve addressed/responded to this comment, without further review

Reply:
Oh, good catch! Thanks